lua/lvim/lsp/manager.lua

```diff
@@ -48,27 +48,39 @@ function M.setup(server_name, user_config)
   if lvim_lsp_utils.is_client_active(server_name) then
     return
   end
+  local installer = require "nvim-lsp-installer"
+  local servers = require "nvim-lsp-installer.servers"
 
   local config = resolve_config(server_name, user_config)
-  local server_available, requested_server = require("nvim-lsp-installer.servers").get_server(server_name)
+  local server_available, requested_server = servers.get_server(server_name)
 
-  local function ensure_installed(server)
-    if server:is_installed() then
-      return true
+  if server_available then
+    -- until server:on_ready() fn exists upstream
+    function requested_server:on_ready(cb)
+      installer.on_server_ready(function(server)
+        if server.name == server_name then
+          cb()
+        end
+      end)
     end
-    if not lvim.lsp.automatic_servers_installation then
-      Log:debug(server.name .. " is not managed by the automatic installer")
-      return false
-    end
-    Log:debug(string.format("Installing [%s]", server.name))
-    server:install()
-    vim.schedule(function()
-      vim.cmd [[LspStart]]
+
+    requested_server:on_ready(function()
+      requested_server:setup(config)
+      local delay_ms = 1000
```

Comment:
This behvaior only exists if the lsp-manager is called from within an ftplugin, even after disabling lunarvim's custom autocmd https://github.com/LunarVim/LunarVim/blob/891cf9944a9f18badffec0c32442610d91f0e96b/lua/lvim/core/autocmds.lua#L10-L12

It works fine otherwise if called normally. I'll open a ticket if I managed to reproduce on a minimum configuration.